plutus-core/plutus-core/test/TypeSynthesis/Spec.hs

```diff
@@ -128,27 +127,26 @@ test_typecheckIllTyped =
             ]
 
 test_typecheckAllFun
-    :: forall fun. (ToBuiltinMeaning DefaultUni fun, Show fun)
+    :: forall fun. (ToBuiltinMeaning DefaultUni fun, Show fun, Show (BuiltinSemanticsVariant fun))
     => String
     -> BuiltinSemanticsVariant fun
-    -> TestTree
-test_typecheckAllFun name semvar
-    = runTestNestedIn ["plutus-core", "test", "TypeSynthesis", "Golden"]
-    . testNested name
+    -> TestNested
+test_typecheckAllFun name semVar
+    = testNestedNamed name (show semVar)
```

Comment:
This now gives better output, before it would just print `DefaultFun` for every semantics variant of `Default`, now it prints the semantics variant instead.